tools/proteore_topgo/topGO.xml

```diff
@@ -0,0 +1,268 @@
+<tool id="topGO" name="Enrichment analysis" version="2021.04.12">
+    <description>(Human, Mouse, Rat)[topGO]</description>
+    <requirements>
+        <requirement type="package">R</requirement>
+        <requirement type="package">r-ggplot2</requirement>
+        <requirement type="package" version="3.8.2">bioconductor-org.hs.eg.db</requirement>
+        <requirement type="package" version="3.8.2">bioconductor-org.mm.eg.db</requirement>
+        <requirement type="package" version="3.8.2">bioconductor-org.rn.eg.db</requirement>
+        <requirement type="package" version="1.62.0">bioconductor-graph</requirement>
+        <requirement type="package" version="1.46.0">bioconductor-annotationdbi</requirement>
+        <requirement type="package" version="3.8.2">bioconductor-go.db</requirement>
+        <requirement type="package" version="2.36.0">bioconductor-topgo</requirement>
+    </requirements>
+    <stdio>
+        <exit_code range="1:" />
+    </stdio>
+    <command><![CDATA[
+
+  Rscript --vanilla $__tool_directory__/topGO_enrichment.R
```

Comment:
please use single-quotes for all path ... e.g. here the tool_directory but also all data-parameters or text-parameters.

See here: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#command-formatting